app/src/Components/Project/ProjectAttachments.tsx

```diff
@@ -17,10 +21,20 @@ const AttachmentDetails = styled.Text`
   font-size: 16px;
   margin-left: 12px;
 `
-
-const ProjectAttachments: FC<ProjectAttachmentsProps> = ({ icon, details }) => {
+/**
+ * Displays the attachment button
+ * @param {url} url of project scope pdf
```

Comment:
yup that makes a lot more sense!

Reply:
Seeing a warning when I add icon and details saying that they are unused parameters...maybe because they're being used within the AttachmentButton JSdocs doesn't read them the say way 🤔